pkg/codegen/testing/test/testdata/dashed-import-schema/nodejs/index.ts

```diff
@@ -5,22 +5,24 @@ import * as pulumi from "@pulumi/pulumi";
 import * as utilities from "./utilities";
 
 // Export members:
-export * from "./provider";
+
+export { ProviderArgs } from "./provider";
+export type Provider = import("./provider").Provider;
+export const Provider: typeof import("./provider").Provider = null as any
+utilities.lazy_load_property(exports, "./provider", "Provider");
+
 
 // Export enums:
 export * from "./types/enums";
 
-// Export sub-modules:
+// Export sub-modules (modContext.genIndex):
```

Comment:
What does `modContext.genIndex` mean here? Accidently debug info included in the commit?

Reply:
Debugging helper, removed.